src/engine/specification.md

```diff
@@ -126,7 +134,7 @@ $ curl https://localhost:8550 \
 
 ## Structures
 
-Fields having `DATA` and `QUANTITY` types **MUST** be encoded according to the [HEX value encoding](https://eth.wiki/json-rpc/API#hex-value-encoding) section of Ethereum JSON-RPC API.
+Encoding of values which have `DATA` and `QUANTITY` types **MUST** follow `bytes` and `uint` schemas from [`base-types.json`](https://github.com/ethereum/execution-apis/blob/main/src/schemas/base-types.json) respectively. Description of encoding notation of these two types may also be found in [HEX value encoding](https://eth.wiki/json-rpc/API#hex-value-encoding) section of Ethereum JSON-RPC API on eth.wiki.
```

Comment:
Same comment here, it feels like we should link to the rendered version.

Reply:
I have updated couple of other places with rendered version of the spec. But I think that this one should either stay as is or the link to `execution-apis` should be removed. In the latter case this part of the spec will refer to eth.wiki.

The reason why I think so is that this particular place describes encoding notation of this pair of types and rendered version doesn't have a dedicated section that would described encoding schemas of primitive types.

Reply:
Lets just get rid of the links entirely, I think they are hurting more than helping:
```suggestion
Encoding of values which have `DATA` type **MUST** be encoded as a hexadecimal string with a `0x` prefix matching the regular expression `^0x(?:[a-fA-F0-9]{2})*$`.

Encoding of values which have `QUANTITY` type **MUST** be encoded as a hexadecimal string with a `0x` prefix and the leading 0s stripped (except for the case of encoding the value `0`) matching the regular expression `^0x(?:0|(?:[a-fA-F1-9][a-fA-F0-9]*))$`.
```

Reply:
Very good suggestion! 👍

Reply:
I have refined wording a bit